app/views/carts/show.html.erb

```diff
@@ -0,0 +1,20 @@
+<h1>Your Cart</h1>
+<table class="table table-striped">
+  <thead>
+    <tr>
+      <th>Item</th>
+      <th>Quantity</th>
+      <th></tr>
+  </thead>
+  <tbody>
+    <% @cart_items.each do |item|%>
+      <tr>
+        <th><%= item[1]%></th>
+        <th> <strong> <%= link_to item[2],item_path(item[0])%> </strong></th>
+        <th> <%= link_to 'Remove', item_cart_path(item_id:item[0]), method: :delete, data: { confirm: 'Are you sure?' } ,style:'color:red',remote: true,class:"delete"%></th>
+      </tr>
```

Comment:
The indexing of object logic is not scalable , unstable and non-maintainable. Please find some other way to accomplish it.

Reply:
Done